cmd/juju/machine/add.go

```diff
@@ -310,7 +302,7 @@ func (c *addCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
-	errs := []error{}
+	errs := make([]error, 0, 10) // avoid extra allocations
```

Comment:
The normal case is for there to be no errors at all. If anything, I think you should change this to `var errs []error`.